tests/unit/lms/services/canvas_api_test.py

```diff
@@ -4,11 +4,9 @@
 from unittest.mock import call, create_autospec, sentinel
 
 import pytest
-from _pytest.mark import param
```

Comment:
Eugh. I need to watch out for this. Auto import in Pycharm appears to detect this private package, which is probably where this is actually defined, above the main exported version.